syntax = "proto3";

import "shared.proto";
import "entity.proto";

package quran;

service Audio {
    rpc CreateAudio (AudioEntity) returns (AudioEntity) {}
    rpc FindAudioById (IDRequest) returns (AudioEntity) {}
    rpc FindAudioByAyahId (IDRequest) returns (AudioEntity) {}
    rpc FindAudioByEditionId (IDRequest) returns (AudioEntity) {}
    rpc FindArabicAudio (FilterRequest) returns (AudioEntity) {}
    rpc FindTranslationAudio (FilterRequest) returns (AudioEntity) {}
}